rllib/agents/__init__.py

```diff
@@ -1,16 +1,8 @@
-from ray.rllib.agents.callbacks import (
-    DefaultCallbacks,
-    MemoryTrackingCallbacks,
-    MultiCallbacks,
-)
-from ray.rllib.agents.trainer import Trainer, with_common_config
-from ray.rllib.agents.trainer_config import TrainerConfig
```

Comment:
This statement is not backward compatible. We need deprecation of all core files under `algorithms.py`

Reply:
See below:
At least importing all Algorithms (under the old "...Trainer" name) AND their default configs (under their old python dict name) is tested now via a new test case that uses the old registry code.